understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
-                        after 3 attempts, and
+                     <li>チャットボットが 3 回試行しても満足のいく応答を提供できない場合は、人間の連絡先を提供する、及び、
                      </li>
                      
-                     <li>can be dismissed with a single interaction, and recalled using a link or button.</li>
+                     <li>1 回の操作で閉じることができ、リンク又はボタンを使用して呼び出すことができる。</li>
                      
                   </ul>
                   
                   
-                  <p>This criterion does not require that a site provide a help mechanism. However, when
-                     help is available:
+                  <p>この達成基準は、サイトがヘルプメカニズムを提供することを要求しない。しかし、ヘルプが利用できる場合:
                   </p>
                   
                   <ul>
                      
-                     <li>People who may have difficulty locating help are more likely to find it and complete
-                        their task.
+                     <li>ヘルプを見つけるのが難しい人でも、ヘルプを見つけて作業を完了する可能性が高くなる。
                      </li>
                      
-                     <li>Users that experience cognitive fatigue or cognitive shut down will be able to reserve
-                        their energy for the task, instead of using it to find support.
+                     <li>認知的疲労又は認知的シャットダウンを経験した利用者は、サポートを見つけるためにエネルギーを使うのではなく、作業のためにエネルギーを蓄えることができる。
                      </li>
                      
-                     <li>Enabling users (especially those with cognitive disabilities) to find solutions while
-                        expressing their question using their own words (for example by interacting with a
-                        chatbot) increases their chances of success for completing a task.
+                     <li>利用者 (特に認知障害のある利用者) が自分の言葉で質問を表現しながら (例えばチャットボットと対話することで) 解決策を見つけられるようにすると、作業の完了に対する成功の可能性が高まる。
                      </li>
                      
                   </ul>
                   
                   
-                  <p>Self help methods beyond the site, such as using internet search to find the contact
-                     information for an organization, can be too difficult. Further, the user's disability
-                     may make it more difficult to find the help available (such as a "contact us" link,
-                     phone number, or support page) if the information is not consistently present within
-                     a few interactions (e.g., displayed in the header, or via a menu). In addition, for
-                     some users with disabilities, struggling to complete a task on a site may cause additional
-                     cognitive challenges when searching for help within the site.
+                  <p>インターネット検索を使用して組織の連絡先情報を見つけるなど、サイトを越えた自己解決の方法は難しすぎる場合がある。さらに、情報が数回の対話内で一貫して存在しない場合 (例えば、ヘッダーに表示される、メニュー経由)、利用者の障害により、利用可能なヘルプ (「お問い合わせ」リンク、電話番号、サポートページなど) を見つけることがより困難になる可能性がある。加えて、障害のある一部の利用者によっては、サイト上で作業を完了するのに苦労することで、サイト内でヘルプを検索するときに、さらなる認知的な問題が発生する可能性がある。
```

Comment:
intaraction は「対話」というよりはシステムとのやりとり、でしょうか。

```suggestion
                  <p>インターネット検索を使用して組織の連絡先情報を見つけるなど、サイトを越えた自己解決の方法は難しすぎる場合がある。さらに、情報が数回のやりとりの中で一貫して存在しない場合 (例えば、ヘッダーに表示される、メニュー経由)、利用者の障害により、利用可能なヘルプ (「お問い合わせ」リンク、電話番号、サポートページなど) を見つけることがより困難になる可能性がある。加えて、障害のある一部の利用者によっては、サイト上で作業を完了するのに苦労することで、サイト内でヘルプを検索するときに、さらなる認知的な問題が発生する可能性がある。
```